zip-0226.rst

```diff
@@ -1,8 +1,313 @@
 ::
 
   ZIP: 226
-  Title: Zcash Shielded Assets - Transfers and Burns
-  Owners: Daniel Bennaroch
-  Status: Reserved
+  Title: Transfer and Burn of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Vivek Arte
+  Status: Draft
   Category: Consensus
+  Created: 2022-05-01
+  License: MIT
   Discussions-To: <https://github.com/zcash/zips/issues/618>
+
+
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Asset: A type of note that can be transferred on the Zcash block chain, identified by the :math:`\mathsf{AssetId}` parameter.
```

Comment:
```suggestion
- Asset: A type of note that can be transferred on the Zcash block chain, identified by the :math:`\mathsf{asset\_desc}` parameter.
```

Reply:
The Asset need not be uniquely identified by the `asset_desc` parameter, it is the combination of the issuance validating key `ik` and the asset description `asset_desc` that is unique. 

@daira had made comments to this effect earlier (mainly [here](https://github.com/zcash/zips/pull/649#discussion_r1042738161), with related comments [here](https://github.com/zcash/zips/pull/649#discussion_r1042777963), [here](https://github.com/zcash/zips/pull/649#discussion_r1042784928), and [here](https://github.com/zcash/zips/pull/649#discussion_r1042787147). 

This seems better than using the Pallas point when we are considering future changes to the curves, but since the `asset_desc` string can be long, we will probably need to hash the `(ik, asset_desc)` tuple using BLAKE2 and use that as the Asset Identifier.